osaca/parser/parser_x86intel.py

```diff
@@ -203,6 +262,33 @@ def process_register(self, operand):
             name=operand["name"],
         )
 
+    def process_memory_address(self, memory_address):
+        """Post-process memory address operand"""
+        # TODO: Use the ptr type.
+        ptr_type = memory_address["ptr_type"]
+        register_expression = memory_address["register_expression"]
+        displacement = register_expression.get(
+            "displacement1",
+            register_expression.get("displacement2", None)
+        )
+        base = register_expression.get("base", None)
+        index = register_expression.get("index", None)
```

Comment:
None is the default `default_value`, see https://pyparsing-docs.readthedocs.io/en/latest/pyparsing.html#pyparsing.ParseResults.get:
```suggestion
        base = register_expression.get("base")
        index = register_expression.get("index")
```

Same elsewhere.